managed/services/telemetry/config.default.yml

```diff
@@ -64,60 +64,168 @@ telemetry:
     query: mongodb_version_info{service_type="mongodb"}
     summary: "Monitored MongoDB services version"
     data:
-      - metric_name: "mongodb_version"
+      - metric_name: "pmm_server_mongodb_version"
         label: "mongodb"
 
   - id: PostgresDBVersion
     source: VM
     query: pg_static{service_type="postgresql"}
     summary: "Monitored PostgreSQL services version"
     data:
-      - metric_name: "postgresql_version"
+      - metric_name: "pmm_server_postgresql_version"
         label: "short_version"
 
-#
+  #
   - id: GrafanaStatTotalUsers
     source: VM
     query: grafana_stat_total_users
     summary: "Total Grafana users"
     data:
-      - metric_name: "grafana_stat_total_users"
+      - metric_name: "pmm_server_grafana_stat_total_users"
         value: 1
 
-# PMM - Usage
+  # PMM - Usage
   - id: PMMUsageNodesCount
     source: PMMDB_SELECT
     query: count(*) AS nodes_count FROM nodes
     summary: "Monitored nodes count"
     data:
-      - metric_name: "pmm_usage_nodes_count"
+      - metric_name: "pmm_server_pmm_usage_nodes_count"
         column: "nodes_count"
 
   - id: PMMUsageServicesCount
     source: PMMDB_SELECT
     query: count(*) AS services_count FROM services
     summary: "Monitored services count"
     data:
-      - metric_name: "pmm_usage_services_count"
+      - metric_name: "pmm_server_pmm_usage_services_count"
         column: "services_count"
 
-# Nodes information
-  - id: NodesInfoAgentsVersions
+  # Nodes information
+  - id: NodesInfoPMMAgent
+    source: PMMDB_SELECT
+    query: version FROM agents WHERE agent_type = 'pmm-agent'
```

Comment:
at the moment version field is available only for pmm-agent, for other agents field will always return an empty value

Reply:
Correct, it will be skipped in this case. I wanted to have a complete list of possible agents.